src/task.py

```diff
@@ -99,15 +99,39 @@ class HIIOSMRasterize(HIITask):
     def __init__(self, *args, **kwargs):
         super().__init__(self, *args, **kwargs)
 
-        self._args = kwargs
-
-        self.osmium_config = self._args.get("osmium_config")
-        self.process_roads = not self._args.get("no_roads")
-        _extent = self._args.get("extent")
+        self.osm_file = kwargs.get("osm_file")
+        self.osm_url = (
+            kwargs.get("osm_url")
+            or os.environ["OSM_DATA_SOURCE"]
+            or "https://ftp.fau.de/osm-planet/pbf/planet-latest.osm.pbf"
+        )
+        self.osmium_text_file = kwargs.get("osmium_text_file") or os.environ.get(
+            "osmium_text_file"
+        )
+        self._working_directory = (
+            kwargs.get("working_dir") or os.environ.get("working_dir") or "/tmp"
+        )
+        _extent = (
+            kwargs.get("extent")
+            or os.environ.get("extent")
+            or ",".join(map(str, HIITask.extent[0] + HIITask.extent[2]))
+        )
         if _extent:
-            self.bounds = [float(c) for c in self._args["extent"].split(",")]
+            self.bounds = [float(c) for c in _extent.split(",")]
         else:
             self.bounds = self.extent[0] + self.extent[2]
+        self.backup_step_data = (
+            kwargs["backup_step_data"]
```

Comment:
this will error if `backup_step_data` hasn't been defined

Reply:
But isn't this the same logic as existed before (see ~244)? Because the argparse uses `store_true` it will be the value on the commandline if exists, otherwise value from env, otherwise False? No?